datalad/consts.py

```diff
@@ -28,6 +29,9 @@
 
 DATASETS_TOPURL = "http://datasets.datalad.org/"
 
+# Centralized deployment
+LOCAL_CENTRAL_PATH = join(expanduser('~'), 'datalad')
```

Comment:
We should use `AppDirs` or sth similar. What is the path to use depends on platform.